### PR TITLE
[compose up] Rewrite exit codes paragraph

### PR DESCRIPTION
**Pull request by shin-**
### Proposed changes

Improved accuracy and fixed grammar

### CODE REVIEW

**Comment by cjerdonek** [comment]
Thanks. I believe `SIGTERM` is [handled the same](https://github.com/docker/compose/blob/f9aaa72c54957ee834141bdf7f7ec9b656f26697/compose/cli/signals.py#L17) as `SIGINT`, so you may want to add that.

**Comment by cjerdonek** [comment]
Also, if the process is interrupted quickly using `SIGINT` (`ctrl` + `C`), the exit code is 1 and not 0.

File: compose/reference/up.md [hunk 1]
```diff
@@ -51,6 +51,6 @@ flag.
 If you want to force Compose to stop and recreate all containers, use the
 `--force-recreate` flag.
 
-If the command encounters an error, it returns `1`.
-If user press `ctrl` + `C`, causing stop of the containers, it returns `0`.
-If user press `ctrl` + `C` two time, forcing kill of containers, it returns `2`.
+If the process encounters an error, the exit code for this command will be `1`.
```
Comment (mstanleyjones) [inline compose/reference/up.md]: Can you replace all the future tense like "will be" with "is"?

**Comment by shin-** [comment]
Updated :+1:

**Pull request merged**